plugins/inputs/http_response/http_response.go

```diff
@@ -49,6 +50,9 @@ var sampleConfig = `
   ## Server address (default http://localhost)
   # address = "http://localhost"
 
+  ## Set proxy (telegraf uses the system wide proxy settings if proxy is not set)
+  # proxy = "http://localhost:8888"
```

Comment:
Let's change this to http_proxy to match the variable name in other plugins.

Reply:
Ok, sounds legit. Did that and will push it right after I tested it.